src/ResourceManager/StorageSync/Commands.StorageSync/Config.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
@afedyashov what is the purpose of this file?

Reply:
Contains configuration settings of our cmdlet. Currently they are embedded into the module (hardcoded). This might change in the future.

Reply:
ConfigSchema was supposed to be JSON schema definition, but we don't use it, will remove.